source/calculus/source/03-AD/08.ptx

```diff
@@ -29,18 +29,252 @@
+
+                    # Create the inner square
+                    inner_square = polygon2d([(x, x), (side_large_square - x, x), (side_large_square - x, side_large_square - x), (x, side_large_square - x)], fill=True, color='lightgray')
+                    inner_square2 = polygon2d([(x,x), (side_large_square - x,x), (side_large_square - x, side_large_square -x), (x, side_large_square - x)], thickness=2, color='darkgray')
+
+                    # Create the small squares (one corner example, replicate for all corners)
+                    small_square_1 = polygon2d([(0, 0), (x, 0), (x, x), (0, x)], fill=True, color='gray', edgecolor="black")
+                    small_square_2 = polygon2d([(side_large_square - x, 0), (side_large_square, 0), (side_large_square, x), (side_large_square - x, x)], fill=True, color='gray', edgecolor="black", )
+                    small_square_3 = polygon2d([(0, side_large_square - x), (x, side_large_square - x), (x, side_large_square), (0, side_large_square)], fill=True, color='gray', edgecolor="black")
+                    small_square_4 = polygon2d([(side_large_square - x, side_large_square - x), (side_large_square, side_large_square - x), (side_large_square, side_large_square), (side_large_square - x, side_large_square)], fill=True, color='gray', edgecolor="black")
+
+                    line_above_small_square = line2d([(6, 8.5), (8, 8.5)], color='black')
+                    side_label_line = line2d([(8.5,6), (8.5,2)], color="black")
+                    lass1 = line2d([(6,8.25), (6,8.75)])
+                    lass2 = line2d([(8,8.25), (8,8.75)])
+                    sll1 = line2d([(8.25,6),(8.75,6)])
+                    sll2 = line2d([(8.25,2),(8.75,2)])
+                    label_side = text("8-2x", (9,4), fontsize=10,color='black')
+                    label_x = text("x", (7,8.75), fontsize=10, color='black')
+
+                    # Combine all parts into one graphic
+                    geometry = Graphics()
+                    geometry += large_square
+                    geometry += inner_square
+                    geometry += inner_square2
+                    geometry += small_square_1
+                    geometry += small_square_2
+                    geometry += small_square_3
+                    geometry += small_square_4
+                    geometry += line_above_small_square
+                    geometry += lass1
+                    geometry += lass2
+                    geometry += label_x
+                    geometry += side_label_line
+                    geometry += sll1
+                    geometry += sll2
+                    geometry += label_side
+
+                    # Set axes and title
+                    geometry.set_axes_range(-1, side_large_square + 1, -1, side_large_square + 1)
+                    geometry.axes(False)
+
+                    # Show the plot
+                    geometry
+                  </sageplot>
```

Comment:
Old:

![image](https://github.com/user-attachments/assets/1881d2e9-993a-4bc9-bc8c-61d04f4f547b)

New:

![image](https://github.com/user-attachments/assets/5990ec65-8347-44d5-ae13-68b16ea8b4be)

I think we should increase the font size in the new one a bit

Reply:
i increased the font size to 15pt- i can go larger if people think it's necessary.